crates/holochain_conductor_api/src/app_interface.rs

```diff
@@ -426,9 +426,14 @@ impl From<AppInfoStatus> for AppStatus {
     }
 }
 
-#[derive(Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize, SerializedBytes)]
+#[derive(Clone, Debug, PartialEq, serde::Serialize, serde::Deserialize, SerializedBytes)]
 pub struct NetworkInfo {
     pub fetch_pool_info: FetchPoolInfo,
+    pub number_of_peers: usize,
```

Comment:
Don't use `usize` or `isize` in apis, we don't want data types that change based on the bit-width of the host system.